singer_sdk/connectors/sql.py

```diff
@@ -477,9 +477,16 @@ def table_exists(self, full_table_name: str) -> bool:
         Returns:
             True if table exists, False if not, None if unsure or undetectable.
         """
+        # pftn -> Parsed Full Table Name [0] = db, [1] = schema, [2] = table
+        pftn: tuple = SQLConnector.parse_full_table_name(
+            self, full_table_name=full_table_name
+        )
+        table_name: str = pftn[2]
+        schema_name: str = pftn[1]
```

Comment:
Since `parse_full_table_name()` always return 3 parts, this should also work:

```py
        db_name, schema_name, table_name = SQLConnector.parse_full_table_name(
            self, full_table_name=full_table_name
        )
```

Or if we know we won't use the db_name part:

```py
        _, schema_name, table_name = SQLConnector.parse_full_table_name(
            self, full_table_name=full_table_name
        )
```

Wdyt?

Reply:
Those are much nicer than what I did.  Just tested the second option and worked great.  also changed to `self` from `SQLConnector`.